### PR TITLE
Add Dygma Raise and update Dygma Shortcut IDs

### PR DESCRIPTION
**Pull request by TheBaronHimself**

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -39,8 +39,8 @@ model01.build.extra_flags={build.usb_flags} -DKALEIDOSCOPE_HARDWARE_H="Kaleidosc
 ##############################################################
 
 shortcut.name=Shortcut
-shortcut.vid.0=0xFEED
-shortcut.pid.0=0x6060
+shortcut.vid.0=0x0x1209
```
Comment (algernon) [inline boards.txt]: Looks like you have a dobule `0x0x` here.

File: boards.txt [hunk 2]
```diff
@@ -61,11 +61,45 @@ shortcut.bootloader.lock_bits=0x2F
 
 shortcut.build.mcu=atmega32u4
 shortcut.build.f_cpu=16000000L
-shortcut.build.vid=0xFEED
-shortcut.build.pid=0x6060
+shortcut.build.vid=0x0x1209
```
Comment (algernon) [inline boards.txt]: Same `0x0x` as before.

**Comment by obra** [comment]
It appears that this request tries to add a PID for another vendor's device.
 http://pid.codes/1209/FEED/
PIDs must be assigned, not asserted. I'm happy to consider a similar PR for a PID code verifiably assigned to Dygma

**Comment by algernon** [comment]
FEED is the one being removed, the new ones being added has been [requested](https://github.com/pidcodes/pidcodes.github.com/pull/282).

**Comment by obra** [comment]
Ok!

File: boards.txt (hunk 1 above)
Comment (TheBaronHimself) [reply boards.txt]: Indeed, thanks. Sorry about the typo.

File: boards.txt (hunk 2 above)
Comment (TheBaronHimself) [reply boards.txt]: Thanks for catching it.

**Comment by TheBaronHimself** [comment]
Can this be reopened, then?

**Comment by obra** [comment]
Algernon, feel free to Merge. I’m going to be off-line for a bit

**Pull request merged**